project.repos

```diff
@@ -2,33 +2,34 @@ repositories:
   # core
   src/wisevision_action_executor:
     type: git
-    url: git@github.com:wise-vision/wisevision_action_executor.git
-    version: main
-  src/wisevision_core:
-    type: git
-    url: git@github.com:wise-vision/wisevision_core.git
-    version: main
-  src/wisevision_lorawan_bridge:
-    type: git
-    url: git@github.com:wise-vision/wisevision_lorawan_bridge.git
-    version: main
+    url: https://github.com/wise-vision/wisevision_action_executor.git
+    version: c_k/dev/switch_to_componnent
   src/wisevision_dashboard:
     type: git
-    url: git@github.com:wise-vision/wisevision-dashboard.git
-    version: main
+    url: https://github.com/wise-vision/wisevision-dashboard.git
+    version: dev
   src/wisevision_data_black_box:
     type: git
-    url: git@github.com:wise-vision/wisevision_data_black_box.git
+    url: https://github.com/wise-vision/wisevision_data_black_box.git
     version: main
   src/wisevision_gps_tools:
     type: git
-    url: git@github.com:wise-vision/wisevision_gps_tools.git
-    version: main
+    url: https://github.com/wise-vision/wisevision_gps_tools.git
+    version: c_k/dev/switch_to_component
   src/wisevision_msgs:
     type: git
-    url: git@github.com:wise-vision/wisevision_msgs.git
+    url: https://github.com/wise-vision/wisevision_msgs.git
     version: main
   src/wisevision_notification_manager:
     type: git
-    url: git@github.com:wise-vision/wisevision_notification_manager.git
-    version: main
+    url: https://github.com/wise-vision/wisevision_notification_manager.git
+    version: c_k/dev/switch_to_component
+  src/wisevision_core:
+    type: git
+    url: https://github.com/wise-vision/wisevision_core.git
+    version: 2501
```

Comment:
Maintain versions consistent if we use the main use it everywhere but as it is related to the specific version it should refer to tags